src/eza/install.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./library_scripts.sh
+
+# nanolayer is a cli utility which keeps container layers as small as possible
+# source code: https://github.com/devcontainers-extra/nanolayer
+# `ensure_nanolayer` is a bash function that will find any existing nanolayer installations,
+# and if missing - will download a temporary copy that automatically get deleted at the end
+# of the script
+ensure_nanolayer nanolayer_location "v0.5.6"
+
+# Example nanolayer installation via devcontainer-feature
+$nanolayer_location \
+    install \
+    devcontainer-feature \
+    "ghcr.io/devcontainers-extra/features/gh-release:1" \
+    --option repo='eza-community/eza' --option binaryNames='eza' --option version="$VERSION"
```

Comment:
```suggestion
    --option repo='eza-community/eza' --option binaryNames='eza' --option version="$VERSION" \
    --option assetRegex='.*(.tar.gz)$'
```

[Tests](https://github.com/devcontainers-extra/features/actions/runs/17007114478/job/48220278252?pr=141) are failing as there are [two matches](https://github.com/devcontainers-extra/features/actions/runs/17007114478/job/48220278252?pr=141#step:6:238):

```
 #15 2.770 nanolayer.installers.gh_release.resolvers.asset_resolver.AssetResolver.AssetResolverError: Too many matches found: ['eza_x86_64-unknown-linux-gnu.tar.gz', 'eza_x86_64-unknown-linux-gnu.zip']
```

Filtering by `*.tar.gz` should fix the problem